docs/state_tests/index.rst

```diff
@@ -0,0 +1,117 @@
+.. _state_tests:
+
+################################################################################
+State Tests
```

Comment:
this tests has been changed drastically new schema is here 
https://github.com/ethereum/EIPs/issues/176  (might be slightly outdated. new field is logs hash)